govcd/api_vcd_versions.go

```diff
@@ -26,6 +26,27 @@ type SupportedVersions struct {
 	VersionInfos `xml:"VersionInfo"`
 }
 
+// map allows you get vCD version from max supported API version
+var vcdVersionToApiVersion = map[string]string{
+	"29.0": "9.0",
+	"30.0": "9.1",
+	"31.0": "9.5",
+	"32.0": "9.7",
+	"33.0": "10.0",
+}
+
+// map allows you get max supported API version from vCD version
+var apiVersionToVcdVersion = map[string]string{
```

Comment:
The names of these two maps are inverted. The names should be switched places, because now the names do opposite of what they say. E.g.:

```
var apiVersionToVcdVersion = map[string]string{
	"9.0":  "29.0",
	"9.1":  "30.0",
	"9.5":  "31.0",
	"9.7":  "32.0",
	"10.0": "33.0",
}

func main() {
	fmt.Println(apiVersionToVcdVersion["29.0"])
}
```

This will not print anything, although one would expect "9.0" to be returned.

CC: @dataclouder @Didainius

Reply:
corrected naming